src/globus_cli/termio/printer.py

```diff
@@ -200,6 +199,9 @@ def __call__(
             sort_json_keys=sort_json_keys,
         )
 
+    def render_table(self, iterable, fields, print_headers=True):
+        print_table(iterable, fields, print_headers)
```

Comment:
This is more or less what we discussed and therefore :+1: from me at its core.

I would like us to put an annotation on it (even if the iterable is `Iterable[Any]`), since this is a method we're exposing to the rest of the CLI code. Can we do that in this PR? (If it proves problematic, I can treat it as a fast-follow.)

Reply:
Added `-> None`